FC-0068 docs: concept cleaning by edunext #11 (#716)
openedx/docs.openedx.org: source/educators/concepts/exercise_tools/OpenResponseAssessments.rst

.. note:: If you include a learner training step, you must also include a peer
   assessment step. The learner training step must come before peer or self
   assessment steps. If you include both peer and self assessment steps, Open

>    edX recommends that you place the peer assessment before the self

Reviewer: Replace Open edX as the subject with Open edX Community.

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

.. _Peer Assessment Step:

Peer Assessment Step
********************

In the peer assessment step, learners review the responses of other learners
in the course. For each response, they select an option for each criterion in
your rubric based on the response. Learners can also provide text feedback, or
comments, on each response.

> If you include both peer and self assessment steps, the Open edX Community

Reviewer 2: ```suggestion
If you include both peer and self assessment steps, it is
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

> you place the peer assessment before the self assessment.

Reviewer: Replace Open edX as the subject with Open edX Community.

> recommends that you place the peer assessment before the self assessment.

Reviewer 2: ```suggestion
recommended that you place the peer assessment before the self assessment.
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

The learner can then select **Continue Assessing Peers** to perform additional
peer assessments.

.. _Self Assessment Step:

Self Assessment Step
********************

In self assessment steps, the learner sees her own response followed by the rubric.
As with peer assessments, the learner evaluates the response using the rubric,
selecting an option for each criterion.

> If you include both peer and self assessments, the Open edX Community

Reviewer 2: ```suggestion
If you include both peer and self assessments, it is
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

> recommends that you include the peer assessment before the self assessment.

Reviewer 2: ```suggestion
recommended that you include the peer assessment before the self assessment.
```

In open response assessments that include only self assessments, the
assignment's final score is equivalent to the self assessment score.

> .. note:: Given the high level of subjectivity in peer assessments, Open edX

Reviewer: Replace Open edX as the subject with Open edX Community.

> .. note:: Given the high level of subjectivity in peer assessments, the Open

Reviewer 2: ```suggestion
.. note:: Given the high level of subjectivity in peer assessments,
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

>    edX Community recommends that you make ORA assignments count towards only a

Reviewer 2: ```suggestion
   it is recommended that you make ORA assignments count towards only a
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

You can allow the **Top Responses** section to show between 1 and 100
responses. Keep in mind, however, that each response might be up to 300 pixels
in height in the list. (For longer responses, learners can scroll to see the

> entire response.) Open edX recommends that you specify 20 or fewer responses to

Reviewer: Replace Open edX as the subject with Open edX Community.

You can allow the **Top Responses** section to show between 1 and 100
responses. Keep in mind, however, that each response might be up to 300 pixels
in height in the list. (For longer responses, learners can scroll to see the

> entire response.) The Open edX Community recommends that you specify 20 or

Reviewer 2: ```suggestion
entire response.) It is recommended that you specify 20 or
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

.. note:: Do not add more than one ORA component in a course unit. Multiple ORA
   assignments in a unit cause errors when learners submit their assessments.

> Open edX suggests that you follow the guidelines and best practices in the

Reviewer: Replace Open edX as the subject with Open edX Community.

.. note:: Do not add more than one ORA component in a course unit. Multiple ORA
   assignments in a unit cause errors when learners submit their assessments.

> The Open edX Community suggests that you follow the guidelines and best

Reviewer 2: ```suggestion
It is suggested that you follow the guidelines and best
```

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0

* Consider using ungraded ORA assignments to generate learner interaction and
  feedback without affecting grades.

Peer Assessments
================

* Set the **Must Grade** number higher than the **Graded By** number to
  minimize the chance that some responses will not be peer assessed. Open edX

>   recommends a setting such as **Must Grade** = 4 and **Graded By** = 3.

Reviewer: Replace Open edX as the subject with Open edX Community.

.. _Prohibited File Extensions:

Prohibited File Extensions
==========================

Learners cannot upload file types that have the following file name extensions
as part of an open response assessment on openedx.org or Open edX Edge. When
you define a set of custom file types for learners to upload with their
responses, you cannot specify these file types. The extensions on this list are
selected and maintained by the development operations team at Open edX, and are
subject to change.

> 

Reviewer: Here, please change the paragraph for the following:
"Learners cannot upload file types with the following file name extensions as part of an Open Response Assessment. When you define a set of custom file types for learners to upload with their responses, you cannot specify them. The extensions on this list are selected and maintained by the development operations team at the Open edX Community and are subject to change."

.. _Prohibited File Extensions:

==========================
Prohibited File Extensions
==========================

Learners cannot upload file types that have the following file name extensions
as part of an open response assessment on edx.org or edX Edge. When you define
a set of custom file types for learners to upload with their responses, you
cannot specify these file types. The extensions on this list are selected and
maintained by the development operations team at edX, and are subject to

> change.

Reviewer 2: Please only remove the _last_ sentence ("The extensions on this list are selected and
maintained by the development operations team at edX, and are subject to
change."), not the whole paragraph.

Author: Done: https://github.com/openedx/docs.openedx.org/commit/72a1b1da6743ab296557c7069f02a23a4d361afa

.. _Prohibited File Extensions:

Prohibited File Extensions
==========================

Learners cannot upload file types with the following file name extensions as

> part of and Open Response Assessment. When you define a set of custom file

Reviewer 2: ```suggestion
part of an Open Response Assessment. When you define a set of custom file
```

types for learners to upload with their responses, you cannot specify them. The
extensions on this list are selected and maintained by the development

> operations team at the Open edX Community and are subject to change.

Author: Updated: https://github.com/openedx/docs.openedx.org/commit/c87cb858aa0bd7b3d365eaab7732784687bfc8c0
